source/MaterialXView/Main.cpp

```diff
@@ -50,7 +50,7 @@ int main(int argc, char* const argv[])
         }
         if (token == "--path" && !nextToken.empty())
         {
-            searchPath = mx::FileSearchPath(nextToken);
+            searchPath.append(mx::FileSearchPath(nextToken));
```

Comment:
This is a good improvement, though I would suggest that we can now remove the explicit cast to mx::FileSearchPath.  The FileSearchPath class should correctly handle "append" with a standard string input.

Reply:
Unfortunately, you can't use a string input argument since it's ambiguous as to which signature it resolves to (the one with FileSearchPath or the one with a FilePath).